lib/jsdom/browser/xhr/xmlhttprequest.js

```diff
@@ -0,0 +1,891 @@
+"use strict";
+
+var HTTP_STATUS_CODES = require("http").STATUS_CODES;
+var deasync;
+try {
+  deasync = require("deasync");
```

Comment:
I replaced it by a spawnSync call but this is really slower than deasync and the http agents cannot be shared between two differents nodejs processes.